docs/topics/logging.txt

```diff
@@ -584,6 +584,12 @@ Logs the SQL queries that are executed during schema changes to the database by
 the :doc:`migrations framework </topics/migrations>`. Note that it won't log the
 queries executed by :class:`~django.db.migrations.operations.RunPython`.
 
+.. versionadded:: 1.10
+
+Messages to this logger have ``params`` and ``sql`` but not ``duration`` in
```

Comment:
rephrase: "params and sql in their extra context (but unlike django.db.backends, not duration)".